docs/docs/self-host/self-host-lightdash-docker-compose.mdx

```diff
@@ -0,0 +1,36 @@
+# Self-host Lightdash using docker-compose
+
+This guide will give you a minimal Lightdash instance running on your local machine. It will not be accessible from 
+the internet, but it will be accessible from your local machine. This is a great way to get started with Lightdash 
+for a proof-of-concept without needing access to kubernetes.
+
+## 1. Clone the Lightdash repository
+
+Clone the Lightdash code to your local machine. This will create a new directory called `./lightdash` (the Lightdash directory).
+
+```bash
+# Clone the Lightdash repo
+git clone https://github.com/lightdash/lightdash
+cd lightdash
+```
+
+## 2. Create containers
+
+You must set the following two environment variables:
+
+- `PGPASSWORD` is the password used for the internal postgres database
+- `LIGHTDASH_SECRET` is the secret used to encrypt data at rest in the database. If you lose this secret, you will not be able to access your data in Lightdash.
+
+```bash
+
+```bash
+LIGHTDASH_SECRET="not very secret" PGPASSWORD="password" \
+  docker-compose -f docker-compose.yml up --detach --remove-orphans
+```
+
+:::info
```

Comment:
There is another very common error for users on Linux not running the daemon... I think we even have an issue about it..